package.json

```diff
@@ -81,14 +82,29 @@
     "tape-catch": "^1.0.4",
     "testron": "^1.2.0",
     "tween.js": "^16.3.5",
-    "uglify-js": "^2.6.1"
+    "uglify-js": "^2.6.1",
+    "module-alias": "^2.0.0",
+    "pre-commit": "^1.2.2",
+    "react-dom": "^15.4.0",
+    "reify": "^0.4.4",
+    "tap-browser-color": "^0.1.2",
+    "tape": "^4.5.1",
+    "tape-catch": "^1.0.4",
+    "webpack": "^2.2.0-rc.3"
   },
   "peerDependencies": {
     "luma.gl": "3.0.0-alpha2",
     "react": "0.14.x - 15.x",
     "react-dom": "0.14.x - 15.x"
   },
   "scripts": {
+    "wp-start": "(cd examples/main && yarn && npm start)",
```

Comment:
Since the `package.json` at the root of the exhibits is not something intended for end-users to copy and get into their project, complexity shouldn't be a real issue, although I agree it might seems a bit obnoxious at first. The important parts are the sub-folders.

We could add this requirement, but we have to keep in mind that a lot of people tend to be quite reluctant to install again another package globally. Do we want to scare them off just because it's easier to support only yarn? Although yarn is the best in my opinion, npm still is the most used package manager out there and the official one (for now ;). Just my 2 satoshis.